app/task_routes.py

```diff
@@ -0,0 +1,97 @@
+from flask import Blueprint
+from app import db
+from app.models.task import Task
+from flask import Blueprint, jsonify, make_response, request, abort
+from sqlalchemy import text
+from datetime import datetime
+import os, requests
+from .routes_helper import validate_model
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix = "/tasks")
+
+@tasks_bp.route("", methods=["POST"])
+def create_task():
+    request_body = request.get_json()
+    if "title" not in request_body or "description" not in request_body:
+        return make_response(jsonify({"details": "Invalid data"}), 400)
+
+    new_task = Task.from_dict(request_body)
+
+    db.session.add(new_task)
+    db.session.commit()
+
+    response_body = dict(task = new_task.to_dict())
+
+    return jsonify(response_body), 201
+
+@tasks_bp.route("", methods=["GET"])
+def get_all_tasks():
+
+    if request.args.get("sort") == "asc":
+        tasks = Task.query.order_by(text("title asc"))
+    elif request.args.get("sort") == "desc":
+        tasks = Task.query.order_by(text("title desc"))
+    else:
+        tasks = Task.query.all()
+
+    tasks_list = [task.to_dict() for task in tasks]
+
+    return jsonify(tasks_list), 200
+
+@tasks_bp.route("/<task_id>", methods=["GET"])
+def get_one_task(task_id):
+    task =validate_model(Task, task_id)
+    response_body = dict(task = task.to_dict())
+
+    return jsonify(response_body), 200
+
+@tasks_bp.route("/<task_id>", methods=["PUT"])
+def update_task(task_id):
+    task =validate_model(Task, task_id)
+    request_body = request.get_json()
+
+    task.title = request_body["title"]
+    task.description =request_body["description"]
```

Comment:
Just like your POST route has error handling, we need that here too in case request_body doesn't have keys "title" or "description" or in case they're misspelled.